### DISCUSSION TITLE
Vueform wants me to use endpoint for forms, but how do i set headers on the call

### DISCUSSION THREAD

**Discussion by YusufTezel** [vueform/vueform#105] [Questions] [answered]
How do i set any header on this request
`<template>
  <Vueform endpoint="/form/submit" method="post">
    <TextElement name="name" label="Name" />
    <TextElement name="email" label="Email" />
  </Vueform>
</template>` 
and, doing the thing here is not an option (a custom header which is dynamic like Authorization can't be here):
export default {
  endpoints: {
    submit: {
      url: '/form/submit',
      method: 'post'
    }
  }
}

**Comment by adamberecz** [comment, marked as answer]
You can set [axios](https://github.com/axios/axios#request-config) headers on config level:
https://vueform.com/reference/configuration#axios

Example:
```js
// vueform.config.js

export default {
  axios: {
    headers: {'X-Requested-With': 'XMLHttpRequest'},
  }
}
```

**Reply by YusufTezel** [reply]
Thanks for your response. I am searching for a dynamic solution, that can help with authorization. Each user will have a different bearer token. Btw I am using axios interceptor to add headers to add headers
